scripts/analyze.sh

```diff
@@ -187,18 +187,29 @@ if [ $? -eq 0 ] && [[ $MODULE != *"BUILD_MODULE_BUFFER"* ]]; then
     >&2 echo Using module: Buffer
     MODULES+=" -DBUILD_MODULE_BUFFER"
 fi
-sensor=$(grep -E Accelerometer\|Gyroscope  $SCRIPT)
+sensor=$(grep -E Accelerometer\|Gyroscope $SCRIPT)
 if [ $? -eq 0 ]; then
     >&2 echo Using module: Sensor
     MODULES+=" -DBUILD_MODULE_SENSOR"
-    echo "CONFIG_SPI=y" >> arc/prj.conf.tmp
+    MODULES+=" -DBUILD_MODULE_SENSOR_TRIGGER"
     echo "CONFIG_SENSOR=y" >> arc/prj.conf.tmp
+    echo "CONFIG_GPIO=y" >> prj.conf.tmp
+    echo "CONFIG_GPIO_QMSI=y" >> prj.conf.tmp
+    echo "CONFIG_GPIO_QMSI_0_PRI=2" >> prj.conf.tmp
+    echo "CONFIG_GPIO_QMSI_1=y" >> prj.conf.tmp
+    echo "CONFIG_GPIO_QMSI_1_NAME=\"GPIO_1\"" >> prj.conf.tmp
+    echo "CONFIG_GPIO_QMSI_1_PRI=2" >> prj.conf.tmp
+    echo "CONFIG_SPI=y" >> arc/prj.conf.tmp
+    echo "CONFIG_SPI_DW_ARC_AUX_REGS=y" >> arc/prj.conf.tmp
+    echo "CONFIG_SPI_DW_INTERRUPT_SEPARATED_LINES=y" >> arc/prj.conf.tmp
```

Comment:
Do the same thing I did for UART:
```
if [ $BOARD = "arduino_101" ]; then
```
This is precisely the reason I changed the script to pass in BOARD from the Makefile.